### PR TITLE
Add Socket-based Java Functional Tests

### PR DESCRIPTION
**Pull request by nidhi1605**
Add Socket-based Java Functional Tests, as per #825 
Signed-off-by: Nidhi Gupta <itsnidhi16@gmail.com>

### CODE REVIEW

File: scripts/travis/openj9-test.sh [hunk 1]
```diff
@@ -5,13 +5,13 @@ cd ../..
...
 docker build -t criu-openj9-ubuntu-test:latest -f scripts/build/Dockerfile.openj9-ubuntu .
-docker run --rm --privileged criu-openj9-ubuntu-test:latest
+docker run -p 49200:49200 --rm --privileged criu-openj9-ubuntu-test:latest
 if [ $? -ne 0 ]; then
 	failures=`echo "$failures ubuntu"`
 fi
 
 docker build -t criu-openj9-alpine-test:latest -f scripts/build/Dockerfile.openj9-alpine .
-docker run --rm --privileged criu-openj9-alpine-test:latest
+docker run -p 49200:49200 --rm --privileged criu-openj9-alpine-test:latest
```
Comment (adrianreber) [inline scripts/travis/openj9-test.sh]: What is the reason to export the port? For local testing? It does not seem to be used here.

Comment (nidhi1605) [reply scripts/travis/openj9-test.sh]: I have removed it.

**Comment by adrianreber** [comment]
I would like to first wait for a resolution of #887 . We should understand why docker is broken.

**Comment by nidhi1605** [comment]
The open-j9 tests pass now. Can somebody please review this PR.

**Comment by adrianreber** [comment]
@nidhi1605 Could you open another pull request to change your alpine tests to install python3 into the container. Like I did with #902 , this should be as simple as replacing `python` with `python3` in `Dockerfile.openj9-alpine`.

**Comment by nidhi1605** [comment]
Opened a PR #903 to switch to python3 in Dockerfile.openj9-alpine.

**Pull request merged**